src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  let objectCopy = { ...state };
```

Comment:
[NAMING] - use proper names for object copy
![image](https://github.com/mate-academy/js_stateful-clones/assets/63243136/5476cd56-2f84-4ca2-a9ca-9c7a70ea1ed9)